testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -128,6 +132,10 @@ Feature: PXE boot a Retail terminal
     And I enter "Terminal branch: example.org" as "description"
     And I click on "Create Group"
     Then I should see a "System group example created." text
+    When I follow "Target Systems"
+    And I check the "proxy" client
+    And I click on "Add Systems"
+    Then I should see a "1 systems were added to example server group." text
```

Comment:
ok, I need to add it to build validation retail tests, i guess

Why do we need this group BTW?

Is it in our documentation?

Reply:
When an image is synced to particular branch server, a group pillar is created to indicate that the image is available for terminals in that branch.
There is a consistency check that the branch server must be part of the group, otherwise it generates an error message in the rhn log:
ERROR com.suse.manager.reactor.messaging.ImageSyncedEventMessageAction - Branch server uyuni-master-pxy.mgr.suse.de is not in group example

I thought it is in the documentation, but now I can't find it, I will take care of that.

Reply:
It is mentioned here https://github.com/uyuni-project/uyuni-docs/blob/master/modules/retail/pages/retail-components.adoc#branch-system-groups
However it is not clear that it must be added manually.

Reply:
OK thanks for the details ; I applied same changes to BV retail tests